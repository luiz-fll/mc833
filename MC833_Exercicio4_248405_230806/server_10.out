remoto: 127.0.0.1:6010
[SERVIDOR] Escutando em 127.0.0.1:6010
127.0.0.1:44668
GET / HTTP/1.0
Host: teste

127.0.0.1:44680
GET / HTTP/1.0
Host: teste

127.0.0.1:44696
GET / HTTP/1.0
Host: teste

127.0.0.1:44698
GET / HTTP/1.0
Host: teste

127.0.0.1:44712
GET / HTTP/1.0
Host: teste

127.0.0.1:44714
GET / HTTP/1.0
Host: teste

127.0.0.1:44730
GET / HTTP/1.0
Host: teste

127.0.0.1:44742
GET / HTTP/1.0
Host: teste

127.0.0.1:44752
GET / HTTP/1.0
Host: teste

