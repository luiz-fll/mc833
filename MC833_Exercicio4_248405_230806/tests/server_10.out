remoto: 127.0.0.1:50010
[SERVIDOR] Escutando em 127.0.0.1:50010
127.0.0.1:34326
GET / HTTP/1.0
Host: teste

127.0.0.1:34338
GET / HTTP/1.0
Host: teste

127.0.0.1:34354
GET / HTTP/1.0
Host: teste

127.0.0.1:34356
GET / HTTP/1.0
Host: teste

127.0.0.1:34370
GET / HTTP/1.0
Host: teste

127.0.0.1:34376
GET / HTTP/1.0
Host: teste

127.0.0.1:34386
GET / HTTP/1.0
Host: teste

127.0.0.1:34398
GET / HTTP/1.0
Host: teste

127.0.0.1:34414
GET / HTTP/1.0
Host: teste

127.0.0.1:34426
GET / HTTP/1.0
Host: teste

